Client/src/Pages/Settings Pages/StudentSettings/ProfileSettings/StudentProfileSettings.js

```diff
@@ -66,6 +67,42 @@ const ProfileInformation = (props) => {
   const url =
     "https://7jdf878rej.execute-api.us-east-2.amazonaws.com/test/users/students/" +
     id;
+  
+  
+  function displaySaveButton() {
+    if (prefName === settings.preferred_name && picture === settings.picture && firstName === settings.first_name && lastName === settings.last_name && email === settings.email && bio === settings.bio && phoneNumber === settings.phone && picture === settings.picture && pronouns === settings.pronouns && linkedIn === getLinkedIn(settings.link)) {
```

Comment:
Just a suggestion but not a necessity.

What about separating each comparison by a new line like
`if (prefName === settings.preferred_name &&`
`picture === settings.picture` ...?

That would make the code more readable.

Reply:
Solved